mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowClient.java

```diff
@@ -55,13 +55,16 @@ public MlflowClient(MlflowHostCredsProvider hostCredsProvider) {
    * @return Run associated with the id.
    */
   public Run getRun(String runId) {
-    URIBuilder builder = newURIBuilder("runs/get").setParameter("run_uuid", runId);
+    URIBuilder builder = newURIBuilder("runs/get")
+      .setParameter("run_uuid", runId)
+      .setParameter("run_id", runId);
     return mapper.toGetRunResponse(httpCaller.get(builder.toString())).getRun();
```

Comment:
BTW, one observation - without changes to the logic here, we won't auto-populate `runId` in the `Run` object returned here (i.e. we just return whatever the server returns to the user), so there's a slight behavior difference between Java & Python (Python's `RunInfo` entity constructor populates `self._run_id` from `run_uuid` if `run_id` is unspecified)

Reply:
I guess the main consequence of not setting `runId` in the returned `Run` here for now is that users of the Java client API will have to check both `runId` and `runUuid` to account for the fact that they could be talking to a new or old server. This is sort of weird, but maybe ok since it's a short-term thing.

Reply:
Actually nvm, the OSS server returns both runId and runUuid. Thus:
* Existing Java code, which uses `run.getRunUuid`, should work w/ the new OSS server
* New Java code, which should use `run.getRunId`, should work w/ the new OSS server.

The only discrepancy then is that new Java code using `run.getRunId` won't work against the old server, but this seems ok - users can code around this initially & once they upgrade their server(s) things should work.

Reply:
Agree with your conclusions. We could also have the Python entity differentiate run_uuid and run_id (allowing different values), but this also has interaction with the file store and potentially sqlalchemy store.